packages/expo-modules-core/ios/AppDelegates/EXReactNativeFactoryDelegate.h

```diff
@@ -2,11 +2,14 @@
 
 #pragma once
 
-#import <ExpoModulesCore/RCTAppDelegateUmbrella.h>
+#import <React_RCTAppDelegate/React-RCTAppDelegate-umbrella.h>
 
 NS_ASSUME_NONNULL_BEGIN
 
-@interface RCTAppDelegate (Recreate)
+NS_SWIFT_NAME(ExpoReactNativeFactoryDelegate)
+@interface EXReactNativeFactoryDelegate : RCTDefaultReactNativeFactoryDelegate
+
+@property (nonatomic, strong, nullable) RCTReactNativeFactory *reactNativeFactory;
```

Comment:
the strong property looks little strange to me and may cause `RCTReactNativeFactory <-> RCTReactNativeFactoryDelegate` cycling retain